src/lib/core/styles/tokens/field/_tokens.scss

```diff
@@ -8,136 +8,58 @@
 @use '../../utils';
 
 $tokens: (
-  text-color: utils.module-val(field, text-color, theme.variable(text-high)),
+  background: utils.module-val(field, background, transparent),
+  tonal-background: utils.module-val(field,tonal-background, theme.variable(surface-container-low)),
+  tonal-background-hover: utils.module-val(field, tonal-background-hover, theme.variable(surface-container-minimum)),
+  filled-background: utils.module-val(field, filled-background, theme.variable(surface-bright)),
```

Comment:
Are we missing `content-color`? The popover icon doesn't seem to override properly in dark theme.

Reply:
This is another thing that wasn't being applied to the default theme